examples/simple-nginx.md

```diff
@@ -54,16 +54,16 @@ Once the pods are created, you can list them to see what is up and running:
 kubectl get pods
 ```
 
-You can also see the replication controller that was created:
+You can also see the deployment that was created:
 
 ```bash
-kubectl get rc
+kubectl get deployment
 ```
 
-To delete the two replicated containers, delete the replication controller:
+To delete the two replicated containers, delete the deployment:
 
 ```bash
-kubectl delete rc my-nginx
+kubectl delete deployment my-nginx
```

Comment:
of if you want to combine run and expose:
```
$ kubectl run my-nginx --image=nginx --replicas=2 --port=80 --expose --service-overrides='{ "spec": { "type": "LoadBalancer" } }'
```

(maybe to complicated for beginner users?)